handler/webhook_test.go

```diff
@@ -0,0 +1,72 @@
+package handler_test
+
+import (
+	"github.com/forjadev/gun-organization/handler"
+	"github.com/gin-gonic/gin"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+)
+
+func TestWebhookReceiverHandle_Success(t *testing.T) {
+	gin.SetMode(gin.TestMode)
+	r := gin.Default()
+	r.POST("/api/v1/webhook", handler.WebhookReceiverHandle)
+	reqPayload := `{
+		"action": "some_action",
+		"scope": "some_scope",
+		"member": {
+			"id": 123,
+			"login": "john_doe",
+			"type": "user",
+			"url": "https://example.com/user/john_doe"
+		},
+		"sender": {
+			"id": 456,
+			"login": "sender_user",
+			"type": "user"
+		},
+		"team": {
+			"id": 789,
+			"name": "some_team",
+			"slug": "some_team_slug"
+		},
+		"organization": {
+			"id": 987,
+			"login": "some_org",
+			"url": "https://example.com/org/some_org"
+		}
+	}`
+	rw := performRequest(r, http.MethodPost, "/api/v1/webhook", reqPayload)
+	assertResponseCode(t, rw.Code, http.StatusOK)
+}
+
+func TestWebhookReceiverHandle_InvalidJSON(t *testing.T) {
+	gin.SetMode(gin.TestMode)
+	r := gin.Default()
+	r.POST("/api/v1/webhook", handler.WebhookReceiverHandle)
+	reqPayload := `{ invalid json payload`
+	rw := performRequest(r, http.MethodPost, "/api/v1/webhook", reqPayload)
+
+	assertResponseCode(t, rw.Code, http.StatusBadRequest)
+}
+
+func TestWebhookReceiverHandle_MethodNotAllowed(t *testing.T) {
+	//TODO: Implementar metodo para retornar o status code MethodNotAllowed(405) pegando do
```

Comment:
You have a placeholder for testing an unsupported method, but it's not implemented yet. We should add that test, right?